### PR TITLE
ADR Split MPS [Rejected]

### PR DESCRIPTION
**Pull request by JasonMarechal25**

### CODE REVIEW

**Review by flomnes — COMMENTED** [review]
Typo in the file name. Other than that, OK !

File: conception/Architecture_decision_records/Split_MPS_files.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Status: Rejected [2022/08/05]
+
+Rejected as not technically viable
+
+## Context
+
+MPS files use to communicate between Antares simulator and Antares xpansion take a lot of disk space. They also take a
+lot of space during xpansion between _problem_generation_ and _benders_ phases.
+MPS files are vastly similar. They share a common part describing the left hand of problems and differ on the part describing the right side of problems
```
Comment (tbittar) [inline conception/Architecture_decision_records/Split_MPS_files.md]: left hand side, right hand side

File: conception/Architecture_decision_records/Split_MPS_files.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Decision
+
+We are proposing to create a single file for the common part and keep several part containing only the diverging part.
```
Comment (tbittar) [inline conception/Architecture_decision_records/Split_MPS_files.md]: several partS

File: conception/Architecture_decision_records/Split_MPS_files.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Consequences
+
+MPS file formatted this way are not considered valid by solvers. We would need to develop parsers mostly reimplementing MPS parsing which would be a
```
Comment (tbittar) [inline conception/Architecture_decision_records/Split_MPS_files.md]: fileS

**Review by tbittar — APPROVED** [review]

**Pull request merged**